mlrun/db/httpdb.py

```diff
@@ -2849,7 +2844,6 @@ def get_marketplace_item(
         """
         path = (f"marketplace/sources/{source_name}/items/{item_name}",)
         params = {
-            "channel": channel,
```

Comment:
💡 Note that few lines under you are using `API` schemas while you are on client side. This will needs to be refactored(not in this PR).

Reply:
FYI It is done all over `httpdb.py` functions, are we going to refactor it in all places, or this is a special case? and if so why?

Reply:
will be needed to be refactored in all places in httpdb